frontend/src/pages/SessionForm/SessionForm.js

```diff
@@ -139,8 +141,9 @@ function SessionForm({ onSendSessionToBackend }) {
     setParticipantDimensions(dimensions);
   };
 
+  console.log("sessionData", sessionData);
   return (
-    <div className="sessionFormContainer">
+    <form className="sessionFormContainer">
```

Comment:
When opening a participant modal, I get the following error:
`Warning: validateDOMNesting(...): <form> cannot appear as a descendant of <form>.`

It seems like the `SessionForm` can include `ParticipantData` components (inside `<form>`), which include a `ParticipantDataModal`, which has its own `<form>`.